WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -13,26 +14,96 @@ final class PointOfSaleDashboardViewModelTests: XCTestCase {
     private var cardPresentPaymentService: MockCardPresentPaymentService!
     private var itemProvider: MockPOSItemProvider!
     private var orderService: POSOrderServiceProtocol!
+    private var totalsViewModel: TotalsViewModel!
+    private var cancellables: Set<AnyCancellable>!
 
     override func setUp() {
         super.setUp()
         cardPresentPaymentService = MockCardPresentPaymentService()
         itemProvider = MockPOSItemProvider()
         orderService = POSOrderPreviewService()
+        totalsViewModel = TotalsViewModel(orderService: orderService,
+                                                  cardPresentPaymentService: cardPresentPaymentService,
+                                                  currencyFormatter: .init(currencySettings: .init()))
         sut = PointOfSaleDashboardViewModel(itemProvider: itemProvider,
                                             cardPresentPaymentService: cardPresentPaymentService,
                                             orderService: orderService,
-                                            currencyFormatter: .init(currencySettings: .init()))
+                                            currencyFormatter: .init(currencySettings: .init())
+                                            , totalsViewModel: totalsViewModel)
+        cancellables = []
     }
 
     override func tearDown() {
         cardPresentPaymentService = nil
         itemProvider = nil
         orderService = nil
         sut = nil
+        cancellables = []
         super.tearDown()
     }
 
+    private func setupViewModelWithExpectationsExpectingAddMoreMatchesSyncingState(paymentState: TotalsViewModel.PaymentState,
```

Comment:
Being helper methods, these could go into an extension. I think there's also no need in the test target for these to be private. Maybe we can also simplify the function names? Since it would seem are specific only to the `AddMoreIsDisabled` property, however both payment state and syncing state are exposed.  Eg:
```
setupViewModel(paymentState: TotalsViewModel.PaymentState, isSyncingOrder: Bool)
```